it/mail/chrome/messenger/messengercompose/sendProgress.dtd

```diff
@@ -1,8 +1,8 @@
 <!--LOCALIZATION NOTE sendprogress.dtd Main UI for Send Message Progress Dialog -->
-<!ENTITY sendWindow.title "Invio messaggi in corso"> 
-<!ENTITY titlePrefixSave "Saving Messages -"> 
-<!ENTITY titlePrefixSend "Sending Messages -"> 
-<!ENTITY status   "Stato:"> 
+<!ENTITY sendWindow.title "Invio messaggi in corso">
+<!ENTITY titlePrefixSave "Salvando messaggi -">
```

Comment:
CORRECTIONS NEEDED. It should be as follows: 

+<!ENTITY titlePrefixSave "Salvataggio messaggi in corso -">